presto-main/src/main/java/io/prestosql/operator/PipelineStats.java

```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
         return internalNetworkInputPositions;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the raw input data size in bytes.
+     *
+     * @return the raw input data size in bytes.
+     */
     @JsonProperty
     public DataSize getRawInputDataSize()
     {
         return rawInputDataSize;
     }
 
+    /**
+     * Returns a property that represents the raw input positions of the PipelineStats.
+     *
+     * @return the raw input positions.
+     */
     @JsonProperty
     public long getRawInputPositions()
     {
         return rawInputPositions;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the processed input data size in bytes.
```

Comment:
It's not clear to me what is this "processed input data". Is any other way to make it clear for the reader?